src/views/create-dataset/CreateDataset.tsx

```diff
@@ -0,0 +1,119 @@
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react'
+import { Alert, Button, Col, Form, Row } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import { useDataset } from '../../sections/dataset/DatasetContext'
+import { DatasetSkeleton } from '../../sections/dataset/DatasetSkeleton'
+import { useLoading } from '../../sections/loading/LoadingContext'
+import styles from '/src/sections/dataset/Dataset.module.scss'
+import { FormInputElement } from '@iqss/dataverse-design-system/src/lib/components/form/form-group/form-element/FormInput'
```

Comment:
If you need to use this type outside of the design-system, please add it to the index file in the design-system and then recompile the package

It should be imported as import { FormInputElement } from '@iqss/dataverse-design-system' instead off using the whole path

Reply:
After converting to Clean Architecture and some cleaner TypeScript I was able to remove this reference completely. Also agree that the dataverse-design-system was a better choice for this import. I'll be more careful moving forward.